src/class/object_columns.php

```diff
@@ -66,17 +66,26 @@ function __construct ()
                                'WHERE c.id = ? ' .
                                'GROUP BY c.id';
 
+        // SQL code for preparing view entry query.
+        // Increase DB limits to allow concatenation of large number of gene/disease IDs.
+        $this->sSQLPreViewEntry = 'SET group_concat_max_len = 200000';
+
         // SQL code for viewing an entry.
         $this->aSQLViewEntry['SELECT']   = 'c.*, ' .
                                            'SUBSTRING_INDEX(c.id, "/", 1) AS category, ' .
                                            'SUBSTRING(c.id, LOCATE("/", c.id)+1) AS colid, ' .
                                            '(a.colid IS NOT NULL) AS active, ' .
                                            'uc.name AS created_by_, ' .
-                                           'ue.name AS edited_by_';
+                                           'ue.name AS edited_by_, ' .
+                                           'GROUP_CONCAT(sc.geneid) as geneids,' .
```

Comment:
You can make LOVD work a bit for you, if you use code like in object_users.php:
`GROUP_CONCAT(sc.geneid ORDER BY sc.geneid SEPARATOR ";") AS _geneids`
Then, you don't have to worry about exploding it yourself, LOVD will do it for you, because of the combination of the underscore in front of the column name, and the semicolon as separator. LOVD will then create $zData['geneids'] for you, already exploded. Also, the ORDER BY may be important. Could you please make these changes for both GROUP_CONCATS?